packages/web-components/src/utils/constants.ts

```diff
@@ -91,3 +91,33 @@ export const IC_BLOCK_COLOR_EXCEPTIONS: IcColorExceptions = {
  */
 export const BLACK_MIN_COLOR_BRIGHTNESS = 136.701;
 export const WHITE_MAX_COLOR_BRIGHTNESS = 130;
+
+// Keyboard characters that do not affect the content of an input control when pressed
+export const IGNORED_KEYBOARD_CHARACTERS = [
+  "Alt",
+  "ArrowDown",
+  "ArrowLeft",
+  "ArrowRight",
+  "ArrowUp",
+  "Control",
+  "End",
+  "Enter",
+  "Escape",
+  "F1",
+  "F2",
+  "F3",
+  "F4",
+  "F5",
+  "F6",
+  "F7",
+  "F8",
+  "F9",
+  "F10",
+  "F11",
+  "F12",
+  "Home",
+  "PageDown",
+  "PageUp",
+  "Shift",
+  "Tab",
```

Comment:
Could we add a few more please?

"CapsLock"
"NumLock"
"Delete"
"Insert"
"Meta" (eg windows key)
"ContextMenu"
"AltGraph"
"AudioVolumeMute"
"AudioVolumeDown"
"AudioVolumeUp"